charts/ingress-nginx/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: ingress-nginx
 # When the version is modified, make sure the artifacthub.io/changes list is updated
 # Also update CHANGELOG.md
-version: 3.19.0
+version: 3.20.0
```

Comment:
You need to remove line 24 and add `- Do not create HPA for defaultBackend if not enabled`

That list is what you see in https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx?modal=changelog

Reply:
I was not aware of that. PR is updated now.